src/applications/mhv-landing-page/utilities/data/index.js

```diff
@@ -161,7 +148,10 @@ const resolveLandingPageLinks = (
     {
       title: HEALTH_TOOL_HEADINGS.MEDICAL_RECORDS,
       icon: 'note_add',
+      introduction:
+        'Our new tool makes it easier to find, print, and download your medical records.',
       links: medicalRecordsLinks,
```

Comment:
Suggest removing the assignment of `HEALTH_TOOL_LINKS.MEDICAL_RECORDS` to a variable and assign directly to the relevant card:

```suggestion
      links: HEALTH_TOOL_LINKS.MEDICAL_RECORDS,
```

Reply:
Makes sense